external-secrets/about-external-secrets-operator.hbs.md

```diff
@@ -1,182 +1,27 @@
-# External Secrets Operator (alpha)
+# External Secrets Operator (beta)
 
->**Caution** External Secrets Operator is in alpha and is intended for
->evaluation and test purposes only. Do not use in a production environment.
+>**Caution** External Secrets plugin available in Tanzu CLI list of plug-ins is in beta 
+and is intended for evaluation and test purposes only. Do not use it in a production environment.
 
 The [External Secrets Operator](https://external-secrets.io) is a Kubernetes
 operator that integrates with external secret management systems, for example,
 Google Secrets Manager and Hashicorp Vault. It reads information from external
-APIs and automatically injects the values into a Kubernetes secret. Starting
-with v1.4.0, Tanzu Application Platform repackages the External Secrets Operator
-v0.6.1 into a Carvel bundle.
+APIs and automatically inject the values into a Kubernetes secret. Tanzu Application Platform makes
```

Comment:
```suggestion
APIs and automatically injects the values into Kubernetes secrets. Tanzu Application Platform makes
```

Reply:
`a secret` - is important in this context.

Reply:
I have updated my suggestion.